### PR TITLE
fix: Remove next.js deprecations

### PR DESCRIPTION
**Pull request by kushthedude**
Refers #985 

Link to the deprecation rules:-
- [Prefetch](https://github.com/zeit/next.js/blob/master/errors/prefetch-true-deprecated.md)
- [Container](https://github.com/zeit/next.js/blob/master/errors/app-container-deprecated.md)
- [Static Directory](https://github.com/zeit/next.js/blob/master/errors/static-dir-deprecated.md)

### CODE REVIEW

**Comment by kushthedude** [comment]
@leecalcote I think we are good to go with this !

**Comment by leecalcote** [comment]
@kushthedude thank you for including links to the deprecation rules. Most helpful.

File: ui/pages/_app.js [hunk 1]
```diff
@@ -354,9 +354,9 @@ class MesheryApp extends App {
     const { isDrawerCollapsed } = this.state;
     return (
       <NoSsr>
-      <Container>
             <Provider store={store}>
                 <Head>
+                <link rel="shortcut icon" href="/ui/static/img/meshery-logo/meshery-logo.svg" />
```
Comment (leecalcote) [inline ui/pages/_app.js:359]: I'll toss in a quick edit to point include `/static` in the path.

**Comment by kushthedude** [comment]
@leecalcote Fixed the lint errors which were due to introduction of new rule of 2 space indent

**Pull request merged**